core/src/main/java/brooklyn/entity/basic/Entities.java

```diff
@@ -270,17 +270,20 @@ public static boolean isTrivial(Object v) {
                 (v instanceof CharSequence&& ((CharSequence)v).length() == 0);
     }
 
+    /**
+     * @deprecated since 0.7
```

Comment:
Can you include in javadoc which method to call (e.g. `@deprecated since 0.7; instead use {@link Sanitizer#sanitize(ConfigBag)}`